lib/src/markdown_processor.dart

```diff
@@ -14,8 +15,23 @@ import 'package:html/parser.dart' show parse;
 import 'package:markdown/markdown.dart' as md;
 
 import 'model.dart';
-
-const bool _emitWarning = false;
+import 'reporting.dart';
+
+const validHtmlTags = const [
+  "a", "abbr", "address", "area", "article", "aside", "audio", "b", "base",
```

Comment:
we should probably comment out the following tags:
- base
- body
- embed
- head
- html
- rb
- rtc

Where did you get the list?